src/entries/popup/hooks/useSearchCurrencyLists.ts

```diff
@@ -66,6 +68,21 @@ const filterBridgeAsset = ({
   asset?.name?.toLowerCase()?.startsWith(filter?.toLowerCase()) ||
   asset?.symbol?.toLowerCase()?.startsWith(filter?.toLowerCase());
 
+/**
+ * @returns a new array of `assets` that don't overlap with `others`
+ */
+function difference(
+  assets: SearchAsset[],
+  others: (SearchAsset | undefined | null)[],
+) {
+  const _others = others.filter(Boolean);
+  return assets.filter((asset) => {
+    return !_others.some((other) =>
+      isLowerCaseMatch(other.uniqueId, asset.uniqueId),
+    );
+  });
+}
```

Comment:
made a more generic function to filter the sections, took the name from [new Set().difference()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/difference)

this changes for example 
```ts
filterAssetsFromBridgeAndAssetToSell(sectionAssets)
``` 
for 
```ts
difference(sectionAssets, [bridgeAsset, assetToSell])
```

I think its more extensible and easier to understand the intention